### PR TITLE
Get Icon data from mapbox instead of fusion tables

### PR DESCRIPTION
**Pull request by nickspeal**
This is on top of #7. I'll merge that first.

Google fusion tables is being deprecated, so we should start to store our data in mapbox instead

### CODE REVIEW

File: public/js/map.js [hunk 1]
```diff
@@ -1,27 +1,28 @@
+import 'whatwg-fetch';
+
+const L = require('mapbox.js');
```
Comment (robotnealan) [inline public/js/map.js:3]: Any idea why it's the norm with mapbox to save the import to a var named `L`?

File: public/js/map.js [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
 const class1Layer = L.mapbox.featureLayer();
 const class2Layer = L.mapbox.featureLayer();
 const class3Layer = L.mapbox.featureLayer();
 const winterLayer = L.mapbox.featureLayer();
-const bikeParkingLayer = L.layerGroup();
-const bikeShopsLayer = L.layerGroup();
-const constructionLayer = L.layerGroup();
+const bikeParkingLayer = L.mapbox.featureLayer();
+const bikeShopsLayer = L.mapbox.featureLayer();
+const constructionLayer = L.mapbox.featureLayer();
```
Comment (robotnealan) [inline public/js/map.js:29]: Got any insight into the key differences between `layerGroup` vs `featureLayer`?

File: public/js/map.js [hunk 3]
```diff
@@ -92,102 +93,76 @@ Promise.all([fetchTrpaData, fetchTruckeeData]).then(combinedData => {
...
+    return `<b>${properties.Name}</b>`;
+  }
 
-    json.rows.forEach((shop) => {
-      const bikeShopIcon = L.divIcon({
-        className: 'bike-shop-icon',
-        iconSize: [24, 24],
-      });
+  const website = properties.Website ? `<a href="${properties.Website}" target="_blank">${properties.Website}</a>` : '';
+  return `<b>${properties.Name}</b><br>LTBC Business Member<br>${properties.Address || ''}<br>${properties['Phone Number'] || ''}<br>${website}`;
```
Comment (robotnealan) [inline public/js/map.js]: `<br>` should be a self-closing tag aka `<br />`

File: public/js/map.js [hunk 4]
```diff
@@ -92,102 +93,76 @@ Promise.all([fetchTrpaData, fetchTruckeeData]).then(combinedData => {
...
 
-function formatBikeShopPopup(shop) {
-  if (shop[1] !== 'LTBC Business Member') {
-    return `<b>${shop[0]}</b>`;
-  }
 
-  const website = shop[5] ? `<a href="${shop[5]}" target="_blank">${shop[5]}</a>` : '';
-  return `<b>${shop[0]}</b><br>LTBC Business Member<br>${shop[2]}<br>${shop[6]}<br>${website}`;
+function createBikeParkingLayer() {
+  const BIKE_PARKING_DATASET_ID = 'ck3gxv6o90p8r2inyo63i5h26';
```
Comment (robotnealan) [inline public/js/map.js]: Would be nice to group these hard-coded IDs together at the top of the file or in a separate constants file we import from, a-la...

```
const DATASET_IDS = {
  BIKE_PARKING: 'ck3gxv6o90p8r2inyo63i5h26',
  CONSTRUCTION: 'ck3gy4qcm2hg42ol93x46deff',
  ...etc
}
```

**Review by robotnealan — APPROVED** [review]

File: public/js/map.js (hunk 1 above)
Comment (nickspeal) [reply public/js/map.js:3]: Heh - yeah I think it's probably from mapbox's origin as an extension to Leaflet JS. 🤷‍♂

File: public/js/map.js (hunk 2 above)
Comment (nickspeal) [reply public/js/map.js:29]: Yeah a layerGroup is an entity to collect several entities in a group. Previously we would add each `Marker` to the map and then group them with the layerGroup. Now we just add a featurelayer and one-shot dump geojson to it.

**Pull request merged**